prow/config/config.go

```diff
@@ -310,6 +314,16 @@ type Branding struct {
 // PubSubSubscriptions maps GCP projects to a list of Topics.
 type PubsubSubscriptions map[string][]string
 
+// GitHubOptions allows users to control how prow application interact with
+// git, github, and github API's.
+type GitHubOptions struct {
+	// LinkURL allows users to override the default github link url for all plugins.
+	// If this option is not set, we use "https://github.com".
+	LinkURL string `json:"link_url,omitempty"`
+
+	// TODO(midnightconman): add APILinkURL variable here.
```

Comment:
Let's not check in TODO comments, just do them later :)

Reply:
removed comment.